### PR TITLE
Add global history.back handler

### PR DESCRIPTION
**Pull request by bfred-it**
`github-injection` happens even when the user navigates in history.
This causes listeners to run on content that has already been updated.

Until now, each feature took care of avoiding its own duplicates, but many were missing or weren't noticeable (e.g. adding the same attribute twice). Most features needed this deduplication, but only half of the current 67 features explicitly checked for duplicates.

Currently broken on master:
- [Single milestone page](https://github.com/babel/babel/milestone/5) (duplicate appears)
- Readme button (duplicate is absolutely positioned on top of the old one)
- A lot of delegated events are added multiple times, nobody would notice it

If you enable logging you will see that most features will not run at all when navigating, yet it will still work.

### CODE REVIEW

**Comment by bfred-it** [comment]
Thanks for testing @sindresorhus! I've had the same experience. I'm gonna merge this and brace for issues :)

File: source/libs/utils.js [hunk 1]
```diff
@@ -8,10 +8,12 @@ import ghInjection from 'github-injection';
...
-// `github-injection` happens even when the user navigates in history
-// This causes listeners to run on content that has already been updated.
-// If a feature needs to be disabled when navigating away,
-// use the regular `github-injection`
+/**
+ *`github-injection` happens even when the user navigates in history
+ * This causes listeners to run on content that has already been updated.
+ * If a feature needs to be disabled when navigating away,
+ * use the regular `github-injection`
+ */
```
Comment (bfred-it) [inline source/libs/utils.js]: @sindresorhus how about this? https://eslint.org/docs/rules/multiline-comment-style

Comment (sindresorhus) [reply source/libs/utils.js]: What exactly do you mean? Do you mean enforcing this style for all comments?

Comment (bfred-it) [reply source/libs/utils.js]: Yep.

Comment (sindresorhus) [reply source/libs/utils.js]: It's up to you. I find it slightly inconvenient as you have to manually manage linebreak and formatting. I haven't found a Sublime plugin that automatically handles that. But then again, you write most of the comments here.

Personally, I use single-line comments for simple things:

```js
// Foo
```

And for long comments, this style:

```js
/*
`github-injection` happens even when the user navigates in history
This causes listeners to run on content that has already been updated.
If a feature needs to be disabled when navigating away,
use the regular `github-injection`
*/
```

It's much easier to format.

Comment (sindresorhus) [reply source/libs/utils.js]: No, I mean a plugin that automatically hard-wraps the comment when written.

**Pull request merged**